test/tests/services/resources-getter.js

```diff
@@ -24,12 +24,12 @@ describe('Service > ResourcesGetter', () => {
           isCompositePrimary: false,
           searchFields: ['amount', 'comment'],
           fields: [
-            { field: '_id', type: 'ObjectId' },
+            { field: '_id', type: 'String' },
```

Comment:
Why is it a `String` now? Does this have an impact?

Reply:
Because ObjectId was a mistake. I don't know who did this but ObjectId never existed in our code base